.github/workflows/manual.yml

```diff
@@ -0,0 +1,17 @@
+name: "Build NixOS/Nixpkgs manuals"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  manuals:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/build-manuals')
```

Comment:
We have two bots @ofborg and @marvin-mk2. Marvin-mk2 also uses the `/` syntax. Probably we should just stick to that. cc @timokau

Reply:
You mean stick to the syntax? I picked it for marvin since github doesn't autocomplete @mentions for gh-apps. Note that there are a few other events too that could contain the text (pull request reviews etc) if you care for completeness. Just issue comments is probably good enough for this purpose though.